docs/build/quickstarts/connecting_to_the_world.md

```diff
@@ -1,34 +1,22 @@
-
-If your function needs to work with some external effects, you need to implement that logic in an effector module that is uniquely identifiable, permissionable and provisioned by participating providers. Not all providers may host a particular effector module. But that providers that do need to serve such an effector module in a consistent manner across heterogenous hosts. 
-
-In order to create a uniquely identifiable effector module, we can use a [CID](https://ipld.io/glossary/#cid) to uniquely identify such a module. Alas, different compiler settings lead to slightly different Wasm compile output and therefore different CIDs. Hence, Fluence is providing a [repo](https://github.com/fluencelabs/fluence-effectors) of permissionable effector modules with their associated CIDs that currently includes a curl and ipfs effector module, respectively.
-
-To be able to use en effector module just as the *curl_adapter* you need to use the version provided by the repo as well as the associated CID (version 1). Neither the `dar` testnet nor the `kras` mainnet peers will accept effector modules that do not hash to the respective CIDs in the repo.
+In order the make managed effects work for a developer, two things are required:
 
-Stay tuned for a more detailed chapter!
+* the host has available the resource you, the developer, wants to access via a Wasm module
+* the host permissions your, the developer's Wasm module utilizing Marine's Mounted Binaries, to access the desired (and hosted) resource
 
+From a high-level perspective then, a manged effect generally requires two components: a hosted resource, such as a binary, adn a Wasm module allowed to access the hosted resource via Mounted Binaries and, effectively, exposing an API to the hosted resource for use by other Wasm modules and, of course, Aqua. 
 
-:::info
-If you need an effector capability currently not offered, contact us in [discord](https://fluence.chat). For development purposes, however, you can use *local network*. 
-:::
+Of course, there are a few more caveats, such as maintaining version compatibility and location (directory) consistency across providers and more. We'll provide a more in-dept presentation, including how to write and provision your own effector(s), shortly. In the meantime, you can utilize two effectors maintained by Fluence and supported by all participating providers: a [curl effector](https://github.com/fluencelabs/curl-effector) and an [IPFS effector](https://github.com/fluencelabs/ipfs-effector). Also see the [example use](https://github.com/fluencelabs/effectors/tree/main/example) of both effectors.
 
 
-## Using The Permissionable cURL Effector
+## Working With The cURL Effector Module
 
 Create a project with Fluence CLI. This time we choose the *minimal* template but stick with the default environment selection, *dar* and name our project *http-enabled*:
 
```

Comment:
should it be quickstart or which of these?

```
? Select template (Use arrow keys)
❯ quickstart
  minimal
  ts
  js
```

Reply:
i used minimal but we can change it to quickstart if that improves dx

Reply:
I'm ok with minimal, it just needs to be stated

Reply:
>This time we choose the *minimal* template ...
isn't that what that does?

Reply:
oh, I'm blind.